src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -583,6 +583,50 @@ services:
     tags:
       - { name: form.type }
 
+  form.type.payment.payment_module_preferences_parent:
+    class: 'PrestaShopBundle\Form\Admin\Improve\Payment\Preferences\PaymentModuleRestrictionsParentType'
+    parent: 'form.type.translatable.aware'
```

Comment:
```suggestion
    abstract: true
    parent: 'form.type.translatable.aware'
```

Reply:
Same as the class, if it's only dedicated as being a parent and can't be used as is the service should defined as an abstrract